src/MCPServer/lib/archivematicaMCP.py

```diff
@@ -59,8 +58,8 @@
 from externals.singleInstance import singleinstance
 from archivematicaFunctions import unicodeToStr
 
-# Print SQL sent to the database interface 
-databaseInterface.printSQL = True
```

Comment:
Hmm, this is going to remove most of the logging in archivematicaMCPClient-<hostname>.log

... that's probably a good thing, but we should look at adding enough logging to usefully replace that.